infrastructure/variables.tf

```diff
@@ -45,3 +45,11 @@ variable "database_sku_capacity" {
   default = "8"
 }
 
+variable "pgsql_sku" {
+  default = "GP_Standard_D2s_v3"
```

Comment:
Remove this variable as default is set in module - https://github.com/hmcts/terraform-module-postgresql-flexible/blob/master/inputs-optional.tf#L22